dev_scripts/convert_params.py

```diff
@@ -0,0 +1,94 @@
+"""Script converting legacy param and json parameter files to hdf5.
+This script downloads the legacy files directly from the GitHub repositories.
+"""
+# Author: George Dang <george_dang@brown.edu.com>
+
+
+import shutil
+import requests
+import os
+from pathlib import Path
+from hnn_core import convert_to_hdf5
+import tempfile
+
+
+def download_folder_contents(owner, repo, path):
```

Comment:
you `pooch` ?

Reply:
https://www.fatiando.org/pooch/latest/

Reply:
If it's stable and reduces the amount of code then perhaps it can be included under the `[dev]` install flag

But considering it would only remove this small function I'm inclined to keep this as is